### PR TITLE
Add install and uninstall targets to Makefile

### PR DESCRIPTION
**Pull request by vrza**
Closes https://github.com/taviso/123elf/issues/66

Supersedes https://github.com/taviso/123elf/pull/20

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,6 +72,24 @@ clean:
 	$(MAKE) -C keymap clean
 
 distclean: clean
-	$(RM) -r orig root share
+	./extract.sh clean
 	./gzip.sh clean
 	./binutils.sh clean
+
+install: bin/123 share/man/man1/123.1
```
Comment (taviso) [inline Makefile:79]: Maybe the install rule should just depend on `all`?

Comment (vrza) [reply Makefile]: Agreed, made the change.

**Comment by vrza** [comment]
Not all files need to be installed, for example from the `sysV386` directory `123` seems to read only the `lotus.bcf` file.

**Comment by taviso** [comment]
Seems okay to me, merged!

**Pull request merged**